### ISSUE TITLE
Fix text overflow display issues

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/sul-requests#1895]
![Screenshot 2023-10-04 at 11 10 58](https://github.com/sul-dlss/sul-requests/assets/111218/2d325b75-9ba2-42a3-bf53-908d28412002)

https://searchworks.stanford.edu/view/L17771

**Comment by jcoyne** [comment]
@alundgard What should this look like when the item description is long and the description is long too (as seen in @cbeer's example)?

**Comment by alundgard** [comment]
A quick fix may be to set `height: fit-content;` on the label element `.form-control` (or a similar solution that makes the row height fit the content). Also seems like an issue for mobile, where every row may be truncated:

<img width="350" alt="mobile-row" src="https://github.com/sul-dlss/sul-requests/assets/2343316/9b99d567-a62e-4760-8ccc-fa17d65ff2af">

**Comment by jcoyne** [comment]
@alundgard any concern that the thing that looks like a table header doesn't appear to encompass the width of this label?
<img width="375" alt="Screenshot 2023-10-04 at 3 35 29 PM" src="https://github.com/sul-dlss/sul-requests/assets/92044/b356b69b-2318-4898-af2b-5b9c3046aee3">

**Comment by alundgard** [comment]
Something that could help a bit with that is setting `text-align: right;` on the span element `.pull-right` (or similar). So that the status message is right-justified with the Status "column":

<img width="356" alt="mobile-row-align-right" src="https://github.com/sul-dlss/sul-requests/assets/2343316/4bafa240-0073-4110-962f-efe17b773f34">

**Comment by jcoyne** [comment]
@alundgard sul-requests is still on bs3. We haven't migrated it yet. I think this may be related  to the switch to folio.

**Comment by dnoneill** [comment]
I don't think this is an issue anymore? The 15th edition is showing up on a separate section of the results in sidebar and is not showing up on requests (which is shouldn't based on sw layout)
